ftw/bob/web/template/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/workflows/+package.fullname_underscore+_workflow/specification.de.txt.bob

```diff
@@ -0,0 +1,75 @@
+[{{{package.fullname}}} workflow]
+
+Initialzustand: Entwurf
+
+Rollen Zuweisung:
+  Gast => Reader
+  Autor => Editor
+  Redaktor => Reviewer
+  Internetbenutzer => Anonymous
+  Verwalter => Site Administrator
+  System Administrator => Manager
+
+
+Sichtbare Rollen:
+  Gast
+  Autor
+  Redaktor
+  Verwalter
+
+
+Allgemein:
+  Ein Autor kann das Gleiche wie ein Gast.
+  Ein Redaktor kann das Gleiche wie ein Autor.
+  Ein Verwalter kann das Gleiche wie ein Redaktor.
+  Ein System Administrator kann das gleiche wie ein Verwalter.
+
+  Ein Verwalter kann jeden Inhalt immer ansehen.
+  Ein Verwalter kann jeden Inhalt bearbeiten.
+  Ein Verwalter kann immer Inhalte hinzufügen.
+  Ein Verwalter kann jeden Inhalt löschen.
+  Ein Verwalter kann immer Rollen zuweisen.
+  Ein Verwalter kann immer Portlets verwalten.
+  Ein Verwalter kann immer veröffentlichen.
+  Ein Verwalter kann immer zurückziehen.
+  Ein Verwalter kann immer zurückweisen.
+  Ein Verwalter kann immer einreichen.
+  Ein Verwalter kann immer Inhalte konfigurieren.
+
+
+Zustandsänderungen:
+# Entwurf
+  einreichen (Entwurf => Eingereicht)
+  veröffentlichen (Entwurf => Veröffentlicht)
+# Eingereicht
+  veröffentlichen (Eingereicht => Veröffentlicht)
+  zurückziehen (Eingereicht => Entwurf)
+  zurückweisen (Eingereicht => Entwurf)
+# Veröffentlicht
+  zurückziehen (Veröffentlicht => Entwurf)
+
+
+Status Entwurf:
+  Ein Gast kann diesen Inhalt ansehen.
```

Comment:
Ich finde `Gast` irgendwie nicht so sinnvoll für Webseiten. In den aktuellsten Projekten haben wir das auch nicht gemacht.
Ich schlage vor, die Rolle `Gast` komplett und ersatzlos zu entfernen.